selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/social/SpaceManagement.java

```diff
@@ -220,6 +279,234 @@ public void saveChangesSpace() {
+        }
+
+        if (managersNames != null) {
+            final String ELEMENT_SPACE_MANAGER = ("//div[@id='spaceManagersList']/ul/li/a[@href and contains(text(),'${user}')]");
+            if (managersNames.get(0) != null && $(byXpath(ELEMENT_SPACE_MANAGER.replace("${user}", managersNames.get(0)))).waitUntil(Condition.appears, Configuration.timeout).isDisplayed()) {
+                assertEquals(managersNames.get(0), $(byXpath(ELEMENT_SPACE_MANAGER.replace("${user}", managersNames.get(0)))).getText());
+            } else {
+                $(byXpath(ELEMENT_SPACE_MANAGER)).waitUntil(Condition.not(Condition.visible), Configuration.timeout);
+            }
+            if (managersNames.get(1) != null && $(byXpath(ELEMENT_SPACE_MANAGER.replace("${user}", managersNames.get(1)))).waitUntil(Condition.appears, Configuration.timeout).isDisplayed()) {
+                assertEquals(managersNames.get(1), $(byXpath(ELEMENT_SPACE_MANAGER.replace("${user}", managersNames.get(1)))).getText());
+            } else {
+                $(byXpath(ELEMENT_SPACE_MANAGER)).waitUntil(Condition.not(Condition.visible), Configuration.timeout);
+            }
+            if (managersNames.get(2) != null && $(byXpath(ELEMENT_SPACE_MANAGER.replace("${user}", managersNames.get(2)))).waitUntil(Condition.appears, Configuration.timeout).isDisplayed()) {
+                assertEquals(managersNames.get(2), $(byXpath(ELEMENT_SPACE_MANAGER.replace("${user}", managersNames.get(2)))).getText());
+            } else {
+                $(byXpath(ELEMENT_SPACE_MANAGER)).waitUntil(Condition.not(Condition.visible), Configuration.timeout);
+            }
+        }
+        if (space != null) {
+            final SelenideElement ELEMENT_SPACE_NAME = $(byXpath("//div[@id='CalendarContainer']/div[@class='eventTitle']/span[@id and contains(text(),'$Space')]".replace("$Space", space)));
+            ELEMENT_SPACE_NAME.waitUntil(Condition.visible,Configuration.timeout);
+            info("Space Name is : " + space);
+            assertEquals(space, ELEMENT_SPACE_NAME.getText());
+        }
+        if (titleEvent != null) {
+            final SelenideElement ELEMENT_TITLE_EVENT_NAME = $(byXpath("//div[@class='pull-left eventSummary']/a[contains(text(),'$TitleEvent')]".replace("$TitleEvent", titleEvent)));
+            ELEMENT_TITLE_EVENT_NAME.waitUntil(Condition.visible,Configuration.timeout);
+            info("Title Event is : " + titleEvent);
+            assertEquals(titleEvent, ELEMENT_TITLE_EVENT_NAME.getText());
+        }
+        if (dateEvent != null) {
+            final String[] ELEMENT_CURRENT_DATE = $(byXpath("//div[@class='currentDateContainer']/center/a")).getText().split(": ");
+            info("Created Event Month is : ");
+            assertEquals(dateEvent.substring(6, 7), ELEMENT_CURRENT_DATE[1].substring(0, 1));
+            info("Created Event day is : ");
+            ArrayList<String> firstNineDays = new ArrayList<>();
```

Comment:
```suggestion
            List<String> firstNineDays = new ArrayList<>();
```

Reply:
Suggestion not applied.

Reply:
It is written in code ArrayList<String> firstNineDays = new ArrayList<>();  
I don't understand the problem exactly